use-timescale/schema-management/foreign-data-wrappers.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Foreign data wrappers
+excerpt: Query PostgreSQL databases within or outside Timescale
+products: [cloud, mst, self_hosted]
+keywords: [hypertables, schemas, alter]
+tags: [change]
+---
+
+# Foreign data wrappers
+
+You use foreign data wrappers (FDWs) to query external data sources from a Timescale Cloud service. These data sources can be one of the following:
+
+- Other Timescale Cloud services.
+- PostgreSQL databases outside of Timescale Cloud.
+
+If you are using [VPC peering][vpc-peering], you can create FDWs in your Customer VPC for a service in Timescale VPC or the same project. However, you can't create FDWs from Timescale VPC to Customer VPC.
+
+FDWs are particularly useful if you manage multiple different Timescale Cloud service types, and need to seamlessly access and merge regular and time-series data.
+
+## Query another data source
+
+You create FDWs with the `postgres_fdw` extension, which is enabled by default. 
+
+<Procedure>
+
+To query another data source, run the following queries in the [SQL editor][sql-editor]:
+
+1. **Create a server:**
+
+   ```sql
+   CREATE SERVER <server-name> 
+   FOREIGN DATA WRAPPER postgres_fdw 
+   OPTIONS (host '<service-ID>.<project-id>.tsdb.cloud.timescale.com', dbname '<database-name>', port '<port-number>');
```

Comment:
Sorry, some last minute comments: `<service-ID>.<project-id>.tsdb.cloud.timescale.com` is correct if another DB is in Timescale Cloud, but can be any correct hostname (or an IPv4 address) otherwise.